src/app/app.component.ts

```diff
@@ -151,7 +152,8 @@ export class CopayApp {
     private bitpayIdProvider: BitPayIdProvider,
     private themeProvider: ThemeProvider,
     private logsProvider: LogsProvider,
-    private dynamicLinksProvider: DynamicLinksProvider
+    private dynamicLinksProvider: DynamicLinksProvider,
+    private newFeatureDataProvider: NewFeatureData
```

Comment:
I'm not completely sure about this. NewFeatureDataProvider is not used in app.component

Reply:
I was using it in my setNetwork method 

ln 233
 ```
[
      this.bitpayProvider,
      this.bitpayIdProvider,
      this.iabCardProvider,
      this.giftCardProvider
    ].forEach(provider => provider.setNetwork(this.NETWORK));
```

 but its causing a dependency injection issue. I have removed that provider from the app component and just changed it to pull network from local storage.